src/amo/components/AddonCompatibilityError/index.js

```diff
@@ -53,8 +54,12 @@ export class AddonCompatibilityErrorBase extends React.Component {
 
     if (reason === INCOMPATIBLE_NOT_FIREFOX) {
       message = i18n.sprintf(i18n.gettext(`You need to
-        <a href="%(downloadUrl)s">download Firefox</a> to install this add-on.`
+        <a href="%(downloadUrl)s">download Firefox</a> to install this
+        add-on.`
```

Comment:
It was. Actually, the tags should really be removed from the translation anyway and made into `%(startLink)s` and `%(endLink)s`, so maybe I'll do those both at the same time. That makes it worth the work of the fuzzy match I think.